### PR TITLE
Feature/add role to user endpoint

### PR DESCRIPTION
**Pull request by lasong**

### CODE REVIEW

File: src/server/routes/index.js [hunk 1]
```diff
@@ -376,6 +378,28 @@ module.exports.sendRole = function(req, res)
...
+{
+  const userId = req.params.userId;
+  const roleId = req.params.roleId;
+
+  Users.userExists(userId).then(exists => {
+    if(exists) {
+      Users.userRoleExists(roleId).then(roleExists => {
+        if(roleExists) {
+          Users.addUserHasRole(userId, roleId).then(userHasRole => res.status('201').json(userHasRole));
+        }
```
Comment (gr4per) [inline src/server/routes/index.js]: when it already is there, created is incorrect. should we rather return 200 ? Maybe 409 is making more sense as the client believes the role is not assigned yet when it fact it is already.

Comment (lasong) [inline src/server/routes/index.js]: I think 200 in this case will do. I would not return 409 cause this means an error, which is actually is not. It shouldn't imo return an error if it already exists. Yes, let's get Chris opinion.

**Comment by kwierchris** [comment]
BTW. Adding users to / Removing users from roles was already available through the update user endpoints.

**Comment by kwierchris** [comment]
THX.

**Pull request merged**